Tests/UITests/LoginUITests.swift

```diff
@@ -0,0 +1,42 @@
+import XCTest
+
+final class LoginUITests: XCTestCase {
+    var sut: WelcomeScreenObject!
+    
+    override func setUp() async throws {
+        continueAfterFailure = false
+        
+        await MainActor.run {
+            sut = WelcomeScreenObject()
+            sut.app.launch()
+        }
+    }
+    
+    override func tearDown() {
+        sut.app.terminate()
+        sut = nil
+    }
+}
+
+extension LoginUITests {
```

Comment:
These tests look great @DylanRJ 

Do we also need a test that confirms the splash screen leads to the welcome screen? (Is that an easy thing to set up...?)

Reply:
It's a really good question, I had a look yesterday around testing the splash screen and saw some conflicting stuff so will maybe have another dig into it today

Reply:
I guess ideally these tests confirm functionality from opening the app to getting to auth, if we can.

Reply:
Yeah from everything we've seen this doesn't seem possible, as it's a system triggered screen.
It might be worth creating a unit test along similar lines to https://github.com/onmyway133/blog/issues/249?
cc. @mcgarj